files/Postgresql_performance.json

```diff
@@ -24,7 +24,8 @@
   "editable": true,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 1,
-  "id": 38,
+  "id": 3,
```

Comment:
are the changee here actually related to the PR?

Reply:
I should have made a note about that, but the current Postgres dashboard has a misconfig where there's a default selection for the database sizes panel, so nothing shows up by default.  I snuck that fix in here.

Reply:
I think it makes sense to pull that into another PR. you are now mixing an rnhancement with a bugfix in the same PT and commit. so reverting the enhancement will introduce a regression. And the dashboard fiy wont show up in the changelog.

Reply:
Agreed, I took it out of this PR.